applicationinsights/channel/SenderBase.py

```diff
@@ -115,7 +115,7 @@ def send(self, data_to_send):
 
         request = HTTPClient.Request(self._service_endpoint_uri, bytearray(request_payload, 'utf-8'), { 'Accept': 'application/json', 'Content-Type' : 'application/json; charset=utf-8' })
         try:
-            response = HTTPClient.urlopen(request)
+            response = HTTPClient.urlopen(request, timeout=10)
```

Comment:
@OsvaldoRosado: I created this[PR](https://github.com/Microsoft/ApplicationInsights-Python/pull/34) to allow configuring SenderBase timeouts. Can you take a look please.